eq-author/src/index.js

```diff
@@ -84,13 +86,15 @@ const headersLink = setContext((_, { headers }) =>
 );
 
 const link = ApolloLink.from([
-  createErrorLink(store),
+  createErrorLink(getStore),
```

Comment:
Is there a danger here that getStore might be called before it is initialised (below)?

Reply:
Actually probably not. It's only used for sending sentry errors (which would require the app to be running)

Reply:
It's one of the things I was hoping to clear up in this pr but we still need the store here for the banner in the error link. Hopefully we can get rid of this after we unpick that last bit.